### ISSUE TITLE
Policy_Compare_String does not support 'not' comparisons

### ISSUE THREAD

**Issue by j-zehnder** [F5Networks/f5-appsvcs-extension#151]
##### Do you already have an issue opened with F5 support? 
No

##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
3.13.1-1
```

##### BIGIP VERSION

```
BIG-IP 14.1.0.6 Build 0.25.9 Engineering Hotfix
```

##### SUMMARY

```
LTM Policy Conditions support not logic in the UI, 'is not', 'does not start with', etc. but the AS3 schema for Policy_Compare_String only supports positive matches.
```

**Comment by dstokesf5** [comment]
Added to our internal product backlog as AT-758. Thank you for reporting this.

**Comment by iluvpcs** [comment]
Curious if we can get an update on this feature request? 

Thank you

**Comment by dstokesf5** [comment]
This feature request should be available in the 3.17 release at the end of January.

**Comment by dstokesf5** [comment]
This should be resolved in the 3.17.0 release. If you are still experiencing a problem, please re-open this issue or create a new one.
